app/src/main/java/ru/chaichuk/hwapp/viewModels/MoviesListViewModel.kt

```diff
@@ -22,10 +24,45 @@ class MoviesListViewModel : ViewModel(){
 
     fun loadMoviesList() {
         if(movies.isEmpty()) {
```

Comment:
not good idea on multithreading 
because your list does not thread-safe
if you want load data one time
you can call loadMovies inside init{} block
or you can check `_mutableMoviesList.value.isEmpty()`

Reply:
forgot to think about that. Now I changed it to "if(_mutableMoviesList.value?.isEmpty() == true)". Next time I will think how to make it better